packit/api.py

```diff
@@ -732,6 +733,37 @@ def sync_status_string(
 """
         return f"'{source_git}' is up to date with '{dist_git}'."
 
+    def check_version_distance(
+        self,
+        current,
+        proposed,
+        target_branch,
+    ) -> None:
+        """Check 'distance' between new and old release
+
+        current: str, already released version for package
+        proposed: str, release we are preparing for package
+        target_branch: str, Fedora branch where release the package
+        """
+        masked_current = re.match(self.package_config.version_update_mask, current)
```

Comment:
I would maybe explicitly return here if the `version_update_mask` is not configured